ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -313,40 +305,42 @@ private boolean createClients(ConnectionStringBuilder dmCsb, ConnectionStringBui
 
     @Test
     void testCreateWithUserPrompt() {
-        ConnectionStringBuilder dmCsb = ConnectionStringBuilder.createWithUserPrompt(System.getenv("DM_CONNECTION_STRING"), null, System.getenv("USERNAME_HINT"));
         ConnectionStringBuilder engineCsb = ConnectionStringBuilder.createWithUserPrompt(System.getenv("ENGINE_CONNECTION_STRING"), null, System.getenv("USERNAME_HINT"));
-        assertTrue(createClients(dmCsb, engineCsb));
+        assertTrue(hasAuthentication(engineCsb));
     }
 
+//    This is an interactive aproach
+//    @Test
```

Comment:
I suggest uncommenting and annotating it as @Ignore("This is an interactive approach. Remove this line to test manually.")